### PR TITLE
Fix geometry bug

### LINKED ISSUE #17: Error after converting Move2_loc to moveStack
The error 'Error in FUN(X[[i]], ...): !anyNA(x) is not TRUE' is returned after using the Move2_loc to moveStack app in an updated version of the Assess Road Interactions workflow that I have been working on. I have done my best to eliminate user error as a cause, but I cannot be certain.

Here is an image of the logs after the app has run and returned the error:
Screenshot 2024-04-24 134421

### PR DESCRIPTION
**Pull request by smthfrmn**
@andreakoelzsch this fixes the bug Ashley was running into with the road crossings app. Again, the vast majority of the changes are just the automated styler (I've commented on the one real change). Let me know if the change seems funky, I ran the function with the real data from Ashley's workflow and it worked and I also ran it with the sample data in the repo and it worked, but tbh I don't actually know much about sf so want to make sure I am not breaking anything.

(I think this fixes #17 )

### CODE REVIEW

File: RFunction.R [hunk 1]
```diff
@@ -1,149 +1,158 @@
...
+
+  # data_ltraj <- as(data,"ltraj") #ltraj trows away projection info, thus dont use it!
+  # data_spdf <- ltraj2sldf(data_ltraj,byid=TRUE)
+
   data_sf <- st_as_sf(data_sldf)
-  #data_sf <- st_as_sf(data)
-  #st_crs(data_sf) <- st_crs(roads)
-  roads <- st_transform(roads,st_crs(data_sf))
-  
+  st_geometry(data_sf) <- "geometry"
```
Comment (smthfrmn) [inline RFunction.R:34]: The only actual, non-style change

Comment (smthfrmn) [reply RFunction.R:34]: @andreakoelzsch does this seem reasonable to you?

Comment (andreakoelzsch) [reply RFunction.R:34]: Hi @smthfrmn, this App is still in move1, so I would rather not rebuild it, as many dependencies migth have changed... Nilanjan had volunteered last time to update this App to move2. I am not sure how far along he is with that. Please would you get in contact and join forces? Thanks! 
@nilanjanchatterjee  FYI

Comment (smthfrmn) [reply RFunction.R:34]: okay per Ashley, we can leave the app as if for now. I'm down to upgrade it to move2 after the workshop. Thanks for reviewing @andreakoelzsch

**Pull request closed without merging**